pmm-app/tests/pages/dashboardPage.js

```diff
@@ -709,6 +709,10 @@ module.exports = {
     navbarLocator: '.navbar-page-btn',
   },
 
+  createAdvancedDataExplorationURL(metricName, time = '&from=now-1m&to=now', nodeName = 'All') {
```

Comment:
please change default value for time and don't make it important for other tests to follow same format, rather just take input as `1m` or `5m` and the rest would always mostly remain same.